Source/santad/SNTApplication.m

```diff
@@ -415,6 +395,27 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
 
     [self stopMetricsPoll];
     [self startMetricsPoll];
+  } else if ([keyPath isEqualToString:NSStringFromSelector(@selector(blockUSBMount))]) {
+    BOOL new = [ change[newKey] boolValue ];
+    BOOL old = [change[oldKey] boolValue];
+
+    if (new != old) {
+      LOGI(@"BlockUSBMount changed: %d -> %d", old, new);
+      self.deviceManager.blockUSBMount = new;
+      if (@available(macos 10.15, *)) [self.deviceManager flushCache];
```

Comment:
Might be worth considering to never cache `AUTH MOUNT` responses, and then we never have to flush all caches on rule updates. Mount operations are already slow, and likely infrequent. Having the ES client handle every one is unlikely to add enough overhead to be noticed. However, flushing the ES cache can definitely have an impact on the system and should be avoided.

Would recommend making `flushCache` an internal-only interface on `SNTDeviceManager.mm`. (maybe even remove the interface altogether since, as currently designed, we subscribe to `AUTH MOUNT` immediately after creating the client which already flushed the cache anyways.

Reply:
Done, removed `flushCache` entirely.